config/config.go

```diff
@@ -102,6 +103,9 @@ type Configuration struct {
 	PriceFloors PriceFloors `mapstructure:"price_floors"`
 }
 
+type Admin struct {
```

Comment:
For my own understanding, why the `Admin` sub-object, instead of just having a `AdminEnabled` boolean in the `Configuration` object?

Reply:
Discussed offline with @SyntaxNode, this comment isn't a concern.